crates/nu-cli/tests/completions/mod.rs

```diff
@@ -258,6 +257,20 @@ fn file_completions() {
         folder(dir.join(".hidden_folder")),
     ];
 
+    #[cfg(windows)]
+    {
+        let separator = '/';
+        let target_dir = format!("cp {dir_str}{separator}");
+        let slash_suggestions = completer.complete(&target_dir, target_dir.len());
+
+        let expected_slash_paths = expected_paths
+            .iter()
+            .map(|s| s.replace('\\', "/"))
+            .collect();
+
+        match_suggestions(expected_slash_paths, slash_suggestions);
```

Comment:
While this ensures that the existing tests still pass after your changes, the existing behavior would have also passed on those tests, if I grok this correctly. Maybe worth adding a few explicit tests to verify your new behavior.

Reply:
Thanks for adding the test @lyuha !

Reply:
Thanks for review!